src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/DF_BaseRecalibrator.java

```diff
@@ -227,6 +326,7 @@ private void ingestLocalInputs(final Pipeline pipeline) throws IOException {
             reads = pipeline.apply(Create.of(readLst).withName("input ingest"));
         }
 
+        // known sites
         List<SimpleInterval> knownSitesLst = new ArrayList<>();
         for (FeatureInput<Feature> vcfSource : BRAC.RAC.knownSites) {
             FeatureDataSource<VariantContext> source = new FeatureDataSource<>(vcfSource.getFeatureFile(), new VCFCodec(), "KnownIntervals");
```

Comment:
Don't assume that the known sites will be passed in in vcf format. Call `FeatureManager.getCodecForFile()` instead of hardcoding a `new VCFCodec()`, make this a `FeatureDataSource<Feature>`, and change the loop below to call the `SimpleInterval` constructor that takes